app/src/components/menu.ts

```diff
@@ -286,6 +289,43 @@ export function createMenu({
     menuTemplate = [editMenu, viewMenu, windowMenu, helpMenu];
   }
 
+  try {
+    const bookmarkConfigPath = path.join(__dirname, '..', 'bookmarks.json');
+    if (fs.existsSync(bookmarkConfigPath)) {
+      const bookmarksConfig = JSON.parse(
+        fs.readFileSync(bookmarkConfigPath, 'utf-8'),
+      );
+      const bookmarksMenu: MenuItemConstructorOptions = {
+        label: bookmarksConfig['menuLabel'],
+        submenu: bookmarksConfig['bookmarks'].map((el) => {
+          if (el['type'] === 'link') {
```

Comment:
We have line space available, the abbreviation is unnecessary and may look weird to someone not used to it; please rename `el` to `bookmark`

Reply:
Done in b418c50709d37f671148e76e3ca15878cb25b4ad